tests/greenheart/test_openmdao.py

```diff
@@ -1,23 +1,28 @@
-from greenheart.tools.optimization.openmdao import HOPPComponent, TurbineDistanceComponent, BoundaryDistanceComponent
-import openmdao.api as om
-from hopp.simulation.hopp_interface import HoppInterface
-from hopp.utilities import load_yaml
-from pytest import approx, raises
+from pytest import approx, raises, fixture
 from pathlib import Path
 import numpy as np
+import copy
+
+import openmdao.api as om
+
+from hopp.simulation import HoppInterface
+from hopp.utilities import load_yaml
+
+from greenheart.tools.optimization.openmdao import HOPPComponent, TurbineDistanceComponent, BoundaryDistanceComponent
+
+import unittest
 
 solar_resource_file = Path(__file__).absolute().parent.parent.parent / "resource_files" / "solar" / "35.2018863_-101.945027_psmv3_60_2012.csv"
 wind_resource_file = Path(__file__).absolute().parent.parent.parent / "resource_files" / "wind" / "35.2018863_-101.945027_windtoolkit_2012_60min_80m_100m.srw"
 floris_input_file = Path(__file__).absolute().parent / "inputs" / "floris_input.yaml"
 hopp_config_filename = Path(__file__).absolute().parent / "inputs" / "hopp_config.yaml"
 
-class TestBoundaryDistanceComponent():
+class TestBoundaryDistanceComponent(unittest.TestCase):
```

Comment:
It would be better to have these use pytest/subtest tools and conventions to be consistent with the rest of the test suite, but this can be done later if we're short on time